src/systemTest/java/com/hivemq/cli/commands/cli/PublishST.java

```diff
@@ -27,21 +28,19 @@
 import org.junit.jupiter.api.Timeout;
 import org.testcontainers.utility.DockerImageName;
 
-import java.io.IOException;
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Set;
-import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
-public class PublishST {
+class PublishST {
 
     private static final @NotNull HiveMQTestContainerExtension hivemq =
             new HiveMQTestContainerExtension(DockerImageName.parse("hivemq/hivemq4"));
 
-    private final @NotNull CLITestExtension cliTestExtension = new CLITestExtension();
+    private final @NotNull MqttCli mqttCli = new MqttCli();
```

Comment:
I think we should name the SystemWrapper different than MqttCli.

Reply:
I moved the whole test changes to #321. So that we can review it independently.